cylc/flow/cfgspec/workflow.py

```diff
@@ -233,7 +232,7 @@
                 Conf('interval', VDR.V_INTERVAL)
 
         with Conf('simulation'):
-            Conf('disable suite event handlers', VDR.V_BOOLEAN, True)
+            Conf('disable workflow event handlers', VDR.V_BOOLEAN, True)
 
         with Conf('environment'):
             Conf('<variable>', VDR.V_STRING)
```

Comment:
These two appear to have snuck in as a regression during #3963, and should be removed. https://github.com/cylc/cylc-flow/issues/3696#issuecomment-713802640 shows that they are meant to be obsolete

Reply:
Good spot, I was actually wondering about that :+1:

Reply:
I got quite confused for a while 🤪

Reply:
The `[environment]` section should be removed too, but can leave to another PR

Reply:
Done.